frontend/packages/ceph-storage-plugin/integration-tests/tests/1-install/installFlow.scenario.ts

```diff
@@ -134,6 +145,148 @@ if (TEST_PLATFORM === Platform.OCP || (TEST_PLATFORM === Platform.OCS && MODE ==
   });
 }
 
+if (
+  TEST_PLATFORM === Platform.OCP ||
+  (TEST_PLATFORM === Platform.OCS && MODE === Mode.ATTACHED_DEVICES)
```

Comment:
Isn't mode being `Mode.ATTACHED_DEVICES` enough?

Reply:
I have kept the conditions in consistent with others.

Reply:
Why is this running in OCP though?

Reply:
Sorry I meant for OCP and OCS with Attached-Devices mode. So yes, just adding MODE for the condition will fulfill the purpose

Reply:
OCP tests run on GCP, this test would not work there. Let's remove the OCP part.